cjdns-core/src/encoding.rs

```diff
@@ -186,8 +186,6 @@ pub fn serialize_forms(encforms: &[EncodingSchemeForm]) -> Result<Vec<u8>, Error
         assert_eq!(accum64, 0u64);
     }
 
-    result_vec.reverse();
-    // println!("[DEBUG] result_vec: {:#x?}", result_vec);
     Ok(result_vec)
 }
 
```

Comment:
In this fn `&Vec` is also not good and unneeded.

I'd suggest to user iterator here (maybe `data: impl Iterator<Item=u8>` or similar to this).